lms/templates/login.html

```diff
@@ -162,6 +159,15 @@ <h3 class="message-title">${_("We couldn't log you in.")} </h3>
         <p class="instructions"> </p>
       </div>
 
+      % if third_party_auth_error:
+        <div role="alert" class="status message third-party-auth-error is-shown" tabindex="-1">
+          <h3 class="message-title">${_("We're sorry, that didn't work.")} </h3>
```

Comment:
@srpearce should this be "An error occurred *when* signing you in to `<platform>`."